files/docker-container-logrotate

```diff
@@ -0,0 +1,9 @@
+/var/lib/docker/containers/*/*.log {
+  rotate 0
+  hourly
+  notifempty
+  nocompress
+  size 20k
```

Comment:
You might need to modify this. https://github.com/coreos/init/blob/master/systemd/system/logrotate.timer